PowerShellGet/PSGet.Resource.psd1

```diff
@@ -70,6 +70,10 @@ ConvertFrom-StringData @'
         PublishModuleLocation=Module '{0}' was found in '{1}'.
         InvalidModuleToPublish=Module '{0}' cannot be published because it does not have a module manifest file. Run New-ModuleManifest -Path <PathName> to create a module manifest with metadata before publishing.         
         MissingRequiredManifestKeys=Module '{0}' cannot be published because it is missing required metadata. Verify that the module manifest specifies Description and Author.
+        
+        InvalidCharactersInPreReleaseString=The PreRelease string contains invalid characters. Please ensure that only characters 'a-zA-Z0-9' and possibly hyphen ('-') at the beginning are in the PreRelease string.

```

Comment:
>PreRelease [](start = 48, length = 10)

During the earlier design review with leads on this Prerelease feature, they have suggested us to use single word without capital R. 
As per the oxford dictionary, it is also a single word -->  [https://en.oxforddictionaries.com/definition/us/prerelease](https://en.oxforddictionaries.com/definition/us/prerelease). 
Please rename 'PreRelease' to 'Prerelease' here and at required places in this feature.